### PR TITLE
Don't require sudo on localhost for updates

### LINKED ISSUE #93: Can't log updates when running without sudo
The "log updated packages" task in bootstrap.yml a) uses become and b) by default tries to write to /var/log/ which requires privileges. Hence packer builds fail if not running as sudo. Could change default to /tmp/?

### PR DESCRIPTION
**Pull request by sjpb**
Fixes #93 so packer builds can run without requiring sudo on localhost

### CODE REVIEW

File: environments/common/inventory/group_vars/all/update.yml [hunk 1]
```diff
@@ -5,4 +5,4 @@ update_state: latest
 update_exclude: omit
 update_disablerepo: omit
 # Log changes during update here on localhost:
-update_log_path: "/var/log/{{ inventory_hostname }}-updates.log"
+update_log_path: "/tmp/{{openhpc_cluster_name}}-{{ inventory_hostname }}-updates.log"
```
[the remark this answers is not in the document]
Comment (sjpb) [reply environments/common/inventory/group_vars/all/update.yml:8]: I don't think that's sufficient as you can also run updates outside of packer. For the packer case maybe we should override it to put it next to the built image?

For the non-packer case I'm not sure where a sensible path is. Maybe we need a `logs/` which defaults to being in the appliance root?

[the remark this answers is not in the document]
Comment (sjpb) [reply environments/common/inventory/group_vars/all/update.yml]: @jovial hopefully fixed by fc45ef5? Think this is a good general pattern

**Review by jovial — APPROVED** [review]
I like it. Could add logs directory to gitignore? or maybe we do want to commit those?

**Comment by sjpb** [comment]
> I like it. Could add logs directory to gitignore? or maybe we do want to commit those?

I had though people /may/ want to so I'm going to leave as-is.

**Pull request merged**